content/en/docs/distributions/minikf/minikf-aws.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Deploy Kubeflow using MiniKF on AWS"
+title = "MiniKF on AWS Marketplace"
 description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via AWS"
```

Comment:
I would remove the parenthesis altogether here and adding "Marketplace":

"Deploy Kubeflow with MiniFK via AWS Marketplace"

Reply:
@cvenets On the description I kept it as is today, but happy to make this change :)